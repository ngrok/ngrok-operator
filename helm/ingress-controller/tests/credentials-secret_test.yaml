suite: test credentials-secret
templates:
- credentials-secret.yaml
tests:
- it: Should match snapshot
  set:
    apiKey: "test-api-key"
    authtoken: "test-authtoken"
  asserts:
  - matchSnapshot: {}
