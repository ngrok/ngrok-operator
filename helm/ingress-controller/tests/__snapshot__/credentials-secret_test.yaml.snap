Should match snapshot:
  1: |
    apiVersion: v1
    data:
      API_KEY: dGVzdC1hcGkta2V5
      AUTHTOKEN: dGVzdC1hdXRodG9rZW4=
    kind: Secret
    metadata:
      name: ngrok-ingress-controller-credentials
    type: Opaque
